src/utils/types/zetkin.ts

```diff
@@ -174,6 +174,7 @@ export interface ZetkinSurvey {
   callers_only: boolean;
   published: string | null;
   expires: string | null;
+  org_access: string;
```

Comment:
I think the type for this could be better, either a union of `'sameorg' | 'suborgs'` or an `enum` with those two values. When it's just typed as `string`, it's easy to misspell the value and not catch it.